### PR TITLE
bpo-30721: Addresses minor fixes of adding ? to redirection syntax

### PR DESCRIPTION
**Pull request by CuriousLearner**
Addresses the minor fixes for PR #2345 

cc @ncoghlan 

https://bugs.python.org/issue30721

### CODE REVIEW

File: Objects/abstract.c [hunk 1]
```diff
@@ -827,7 +827,7 @@ binary_op(PyObject *v, PyObject *w, const int op_slot, const char *op_name)
             PyErr_Format(PyExc_TypeError,
                 "unsupported operand type(s) for %.100s: "
                 "'%.100s' and '%.100s'. Did you mean \"print(<message>, "
-                "file=<output_stream>)\"",
+                "file=<output_stream>)?\"",
```
Comment (ncoghlan) [inline Objects/abstract.c]: The question mark should be outside the quote marks, since it isn't part of the suggested Python code.

**Review by ncoghlan — CHANGES_REQUESTED** [review]
While English does in general suggest that trailing punctuation should go inside quotation marks (although I've never personally understood that guideline), in this case it should definitely be outside, as what's inside is expected to be a Python expression, not English text.

**Comment by CuriousLearner** [comment]
I didn't expect the Spanish Inquisition!

**Review by ncoghlan — APPROVED** [review]

**Comment by CuriousLearner** [comment]
Thanks @ncoghlan :)

**Pull request merged**